pkg/webhooks/errors_test.go

```diff
@@ -91,37 +96,34 @@ func TestIsAdmissionCantConnectAPIServer(t *testing.T) {
 
 	for _, tc := range testCases {
 		t.Run(tc.testName, func(t *testing.T) {
-			if got := IsAdmissionCantConnectAPIServer(tc.err); got != tc.want {
-				t.Errorf("Check connection to API Server error message. Expected: %t ; Got: %t", tc.want, got)
-			}
+			got := IsAdmissionCantConnectAPIServer(tc.err)
+			require.Equal(t, tc.want, got)
 		})
 	}
 }
 
 func TestIsAdmissionOutOfRangeReplicationFactor(t *testing.T) {
 	kafkaTopic := banzaicloudv1alpha1.KafkaTopic{ObjectMeta: metav1.ObjectMeta{Name: "test-KafkaTopic"}}
 	var fieldErrs field.ErrorList
-	fieldErrs = append(fieldErrs, field.Invalid(field.NewPath("spec").Child("replicationFactor"), "-2", outOfRangeReplicationFactorErrMsg))
+	fieldErrs = append(fieldErrs, field.Invalid(field.NewPath("spec").Child("replicationFactor"), int32(-2), outOfRangeReplicationFactorErrMsg))
```

Comment:
**Question**: Why were these type changes necessary? Or were they just linter suggestions?

Reply:
No, that is fine, I just wasn't sure whether this was required by compiler, suggested by linter or had another reasoning behind, like what you explained.
Makes sense.